internal/clusteroperator/release.go

```diff
@@ -0,0 +1,16 @@
+package clusteroperator
+
+import "os"
+
+const (
+	defaultVersionValue    = "0.0.1-snapshot"
+	releaseVersionVariable = "RELEASE_VERSION" // OpenShift's env variable for defining the current release
+)
+
+func GetReleaseVariable() string {
+	v := os.Getenv(releaseVersionVariable)
+	if v == "" {
+		return defaultVersionValue
```

Comment:
I wonder if us not being able to find the `RELEASE_VERSION` env var should cause an error or not like it does in [OLM today](https://github.com/operator-framework/operator-lifecycle-manager/blob/b2086bd2c5eb9b60370869757c51fa7908d2cde0/pkg/controller/operators/openshift/helpers.go#L214). Is there a situation where this value not being present is okay and we can set a default?

Reply:
I did this so it's a bit easier to run locally without having to explicitly set the environment variable. This seems sufficient for now, but it's probably worth revisiting this decision over time.